src/main/java/com/virtuslab/using_directives/custom/utils/TokenUtils.java

```diff
@@ -32,14 +32,10 @@ public static boolean isIdentifier(Tokens token) {
   }
 
   public static boolean isValidUsingDirectiveStart(Tokens token, Settings settings) {
-
-    boolean res =
-        (settings.isAllowStartWithoutAt()
-                && (token == Tokens.USING
-                    || (settings.isAllowRequire() && token == Tokens.REQUIRE)))
-            || token == Tokens.ATUSING
-            || (settings.isAllowRequire() && token == Tokens.ATREQUIRE);
-    return res;
+    return (settings.isAllowStartWithoutAt()
```

Comment:
Can you try changing the formatting, so that it will be easier to read?

Reply:
It's formatted using Spotless

Reply:
ok, nvm then